plugin/pkg/scheduler/scheduler.go

```diff
@@ -92,6 +97,11 @@ func (s *Scheduler) scheduleOne() {
 		glog.V(1).Infof("Failed to schedule: %+v", pod)
 		s.config.Error(pod, err)
 		s.config.Recorder.Eventf(pod, api.EventTypeWarning, "FailedScheduling", "%v", err)
+		s.config.PodConditionUpdater.Update(pod, &api.PodCondition{
+			Type:   api.PodScheduled,
+			Status: api.ConditionFalse,
+			Reason: "Unschedulable",
```

Comment:
Agree with this, it's better to let the user understand why the pod is failed to schedule.